docs/discover/context.asciidoc

```diff
@@ -1,90 +1,66 @@
-data] usage. Common examples for suitable fields include log line numbers,
-monotonically increasing counters and high-precision timestamps.
---
 
+[role="screenshot"]
 image::images/Discover-ContextView.png[Context View]
 
-NOTE: The number of documents displayed by default can be configured
-via the `context:defaultSize` setting in <<advanced-options, *Management >
-Advanced Settings*>>.
-
 [float]
-[[change-context-size]]
-=== Changing the Context Size
-
-You can change the number documents displayed before and after the anchor
-document independently.
-
-To increase the number of displayed documents that are newer than the anchor
-document, click the *Load 5 more* button above the document list or enter the
-desired number into the input box right of the button.
-
-image::images/Discover-ContextView-SizePicker-Newer.png[]
-{nbsp}
-
-To increase the number of displayed documents that are older than the anchor
-document, click the *Load 5 more* button below the document list or enter the
-desired number into the input box right of the button.
+[[filter-context]]
+=== Filter the context
 
-image::images/Discover-ContextView-SizePicker-Older.png[]
-{nbsp}
+The
+filters you applied in *Discover* are carried over to the Context view. Pinned filters remain active, while normal
+filters are copied in a disabled state. You can re-enable these filters to
+refine your context view.
 
-NOTE: The default number of documents loaded with each button click can be
-configured via the `context:step` setting in <<advanced-options, *Management >
-Advanced Settings*>>.
+If the Context view contains a large number of documents not related to the event under
+investigation, you can use filters to restrict the documents to
+display.
 
 [float]
-[[filter-context]]
-=== Filtering the Context
-
-Depending on how the documents are partitioned into index patterns, the context
-view might contain a large number of documents not related to the event under
-investigation. In order to adapt the focus of the context view to the task at
-hand, you can use filters to restrict the documents considered by Kibana for
-display in the context view.
-
-When switching from the discover view to the context view, the previously
-applied filters are carried over. Pinned filters remain active while normal
-filters are copied in a disabled state. You can selectively re-enabled them to
-refine your context view.
+[[change-context-size]]
+=== Change the number of surrounding documents
 
-New filters can be added via the *Add a filter* link in the filter bar, by
-clicking the filter icons appearing when hovering a field, or by expanding
-documents and clicking the filter icons in the table.
+By default, the five newer and older
+documents are listed. To increase the number of documents that surround the anchor document,
+click *Load*.  Five documents are added with each click, but you can change this number in Advanced Settings.
```

Comment:
I would add a period after `click`, then remove `, but you can change this number in Advanced Settings.`